sdk/monitor/monitor-opentelemetry-exporter/samples-dev/GRPCServer.ts

```diff
@@ -0,0 +1,44 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * A gRPC server that receives requests on sample server port.
+ *
+ * @summary a gRPC server
```

Comment:
Extend this summary: a gRPC server that does/shows what with respect to the library and its most important scenarios?